gatsby-node.js

```diff
@@ -62,3 +66,28 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
         });
     }
 }
+
+exports.onPostBuild = async () => {
+    const publicPath = path.join(__dirname, 'public');
+    const files = sync(`${publicPath}/**/*.{html,js}`);
+    const hash = createHash(`sha256`)
+        .update(Date.now().toString())
+        .digest(`hex`);
```

Comment:
Is there a reason to make this hash instead of using plan datetime? 
I mean just use `const hash = Date.now().toString()`, and no additional library required.